masonite/drivers/BaseUploadDriver.py

```diff
@@ -0,0 +1,26 @@
+import os
+from masonite.exceptions import ExtensionException
+
+
+class BaseUploadDriver(object):
+    """
+    Class base for upload drivers
+    """
+    def __init__(self):
+        # this will accept all file types
+        self.accept_file_types = None
+
+    def accept(self, *args, **kwargs):
+        """
+        Set files to accept
```

Comment:
Change this to `Set file types to accept` since it's the file types we want to accept

Reply:
I do not understand, why text replaced it?

Reply:
Sorry, I read it wrong